### ISSUE TITLE
mail_view stopped working with 2.0.3

### ISSUE THREAD

**Issue by kenn** [basecamp/mail_view#63]
Starting 2.0.3, I'm seeing the following error by accessing `/mail_view`. Thoughts?

```
Errno::ENOENT - No such file or directory - /Users/kenn/.rvm/gems/ruby-2.0.0-p247/gems/mail_view-2.0.3/lib/mail_view/index.html.erb:
  tilt (1.4.1) lib/tilt/template.rb:91:in `read_template_file'
  tilt (1.4.1) lib/tilt/template.rb:68:in `block in initialize'
  tilt (1.4.1) lib/tilt/template.rb:69:in `initialize'
  tilt (1.4.1) lib/tilt.rb:61:in `new'
  mail_view (2.0.3) lib/mail_view.rb:80:in `index_template'
  mail_view (2.0.3) lib/mail_view.rb:32:in `call'
  mail_view (2.0.3) lib/mail_view.rb:20:in `call'
  actionpack (4.0.2) lib/action_dispatch/journey/router.rb:71:in `block in call'
  actionpack (4.0.2) lib/action_dispatch/journey/router.rb:59:in `call'
  actionpack (4.0.2) lib/action_dispatch/routing/route_set.rb:680:in `call'
  newrelic_rpm (3.7.0.177) lib/new_relic/rack/error_collector.rb:50:in `call'
  newrelic_rpm (3.7.0.177) lib/new_relic/rack/agent_hooks.rb:28:in `call'
  newrelic_rpm (3.7.0.177) lib/new_relic/rack/browser_monitoring.rb:23:in `call'
  newrelic_rpm (3.7.0.177) lib/new_relic/rack/developer_mode.rb:42:in `call'
  request_store (1.0.5) lib/request_store/middleware.rb:9:in `call'
  rack (1.5.2) lib/rack/etag.rb:23:in `call'
  rack (1.5.2) lib/rack/conditionalget.rb:25:in `call'
  rack (1.5.2) lib/rack/head.rb:11:in `call'
  actionpack (4.0.2) lib/action_dispatch/middleware/params_parser.rb:27:in `call'
  actionpack (4.0.2) lib/action_dispatch/middleware/flash.rb:241:in `call'
  rack (1.5.2) lib/rack/session/abstract/id.rb:225:in `context'
  rack (1.5.2) lib/rack/session/abstract/id.rb:220:in `call'
  actionpack (4.0.2) lib/action_dispatch/middleware/cookies.rb:486:in `call'
  activerecord (4.0.2) lib/active_record/query_cache.rb:36:in `call'
  activerecord (4.0.2) lib/active_record/connection_adapters/abstract/connection_pool.rb:626:in `call'
  activerecord (4.0.2) lib/active_record/migration.rb:369:in `call'
  actionpack (4.0.2) lib/action_dispatch/middleware/callbacks.rb:29:in `block in call'
  activesupport (4.0.2) lib/active_support/callbacks.rb:373:in `_run__867493693933363244__call__callbacks'
  activesupport (4.0.2) lib/active_support/callbacks.rb:80:in `run_callbacks'
  actionpack (4.0.2) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (4.0.2) lib/action_dispatch/middleware/reloader.rb:64:in `call'
  actionpack (4.0.2) lib/action_dispatch/middleware/remote_ip.rb:76:in `call'
  airbrake (3.1.14) lib/airbrake/rails/middleware.rb:13:in `call'
  better_errors (1.0.1) lib/better_errors/middleware.rb:84:in `protected_app_call'
  better_errors (1.0.1) lib/better_errors/middleware.rb:79:in `better_errors_call'
  better_errors (1.0.1) lib/better_errors/middleware.rb:56:in `call'
  actionpack (4.0.2) lib/action_dispatch/middleware/debug_exceptions.rb:17:in `call'
  actionpack (4.0.2) lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
  railties (4.0.2) lib/rails/rack/logger.rb:38:in `call_app'
  railties (4.0.2) lib/rails/rack/logger.rb:20:in `block in call'
  activesupport (4.0.2) lib/active_support/tagged_logging.rb:67:in `block in tagged'
  activesupport (4.0.2) lib/active_support/tagged_logging.rb:25:in `tagged'
  activesupport (4.0.2) lib/active_support/tagged_logging.rb:67:in `tagged'
  railties (4.0.2) lib/rails/rack/logger.rb:20:in `call'
  quiet_assets (1.0.2) lib/quiet_assets.rb:18:in `call_with_quiet_assets'
  actionpack (4.0.2) lib/action_dispatch/middleware/request_id.rb:21:in `call'
  rack (1.5.2) lib/rack/methodoverride.rb:21:in `call'
  rack (1.5.2) lib/rack/runtime.rb:17:in `call'
  activesupport (4.0.2) lib/active_support/cache/strategy/local_cache.rb:83:in `call'
  rack (1.5.2) lib/rack/lock.rb:17:in `call'
  actionpack (4.0.2) lib/action_dispatch/middleware/static.rb:64:in `call'
  rack (1.5.2) lib/rack/sendfile.rb:112:in `call'
  airbrake (3.1.14) lib/airbrake/user_informer.rb:16:in `_call'
  airbrake (3.1.14) lib/airbrake/user_informer.rb:12:in `call'
  railties (4.0.2) lib/rails/engine.rb:511:in `call'
  railties (4.0.2) lib/rails/application.rb:97:in `call'
  rack (1.5.2) lib/rack/content_length.rb:14:in `call'
  thin (1.6.1) lib/thin/connection.rb:82:in `block in pre_process'
  thin (1.6.1) lib/thin/connection.rb:80:in `pre_process'
  thin (1.6.1) lib/thin/connection.rb:55:in `process'
  thin (1.6.1) lib/thin/connection.rb:41:in `receive_data'
  eventmachine (1.0.3) lib/eventmachine.rb:187:in `run'
  thin (1.6.1) lib/thin/backends/base.rb:73:in `start'
  thin (1.6.1) lib/thin/server.rb:162:in `start'
  rack (1.5.2) lib/rack/handler/thin.rb:16:in `run'
  rack (1.5.2) lib/rack/server.rb:264:in `start'
  railties (4.0.2) lib/rails/commands/server.rb:84:in `start'
  zeus (0.13.3) lib/zeus/rails.rb:158:in `server'
  zeus (0.13.3) lib/zeus.rb:116:in `block in command'
  zeus (0.13.3) lib/zeus.rb:103:in `command'
  zeus (0.13.3) lib/zeus.rb:80:in `block (3 levels) in go'
  zeus (0.13.3) lib/zeus.rb:80:in `block (2 levels) in go'
  zeus (0.13.3) lib/zeus.rb:73:in `block in go'
  zeus (0.13.3) lib/zeus.rb:62:in `go'
  zeus (0.13.3) lib/zeus.rb:78:in `block (3 levels) in go'
  zeus (0.13.3) lib/zeus.rb:78:in `block (2 levels) in go'
  zeus (0.13.3) lib/zeus.rb:73:in `block in go'
  zeus (0.13.3) lib/zeus.rb:62:in `go'
  zeus (0.13.3) lib/zeus.rb:78:in `block (3 levels) in go'
  zeus (0.13.3) lib/zeus.rb:78:in `block (2 levels) in go'
  zeus (0.13.3) lib/zeus.rb:73:in `block in go'
  zeus (0.13.3) lib/zeus.rb:62:in `go'
  -e:1:in `<main>'
```

**Comment by jeremy** [comment]
Messed up the packaged gem - sorry! Fixed @ a8f5056453dd7acc4c23e889a3a0cefb75fb28c3 and pushed 2.0.4.

**Comment by kenn** [comment]
Thanks Jeremy!
